tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,358 @@
+
+</div>
+</SimpleTab>
+
+## Create and Reset Dedicated database password
+
+When you connect to your cluster for the first time,You should click **Security Setting** to automatically generate a root password with a length of 16 characters, including numbers, uppercase and lowercase characters, and special characters. 
+
+## Root Digital certificate management of Dedicated Tier 
+
+### Root Digital certificate‘s issuance and validity  
+
+TiDB Dedicated Tier uses certificates from [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Dedicated Tier clusters. Usually, the root certificate of ACM are stored securely in AWS managed hardware security modules (HSMs) that adhere to [FIPS 140-2 Level 3](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3139) security standards. 
+
+However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Dedicated Tier cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
+
+### Root Digital certificate‘s default path 
+
+In different operating systems, the storage path of the root certificate is as follows：
+
+**MacOS**
+
+```
+/etc/ssl/ca_root.pem
+```
+
+**Debian / Ubuntu / Arch**
+
+```
+/etc/ssl/certs/ca_root.crt
+```
+
+**RedHat / Fedora / CentOS / Mageia**
+
+```
+/etc/pki/tls/certs/ca_root.crt
+```
+
+**Alpine**
+
+```
+/etc/ssl/ca_root.pem
+```
+
+**OpenSUSE**
+
+```
+/etc/ssl/ca_root.pem
+```
+
+**Windows**
+
+Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, to specify the CA root path on Windows, take the following steps:
+
+## FAQs
+
+### Which TLS versions should be supported to connect to my TiDB Dedicated Tier cluster?
+
+TiDB Dedicated cluster only supports TLS 1.2 or TLS 1.3, and does not support TLS 1.0 and TLS 1.1 versions for security reasons. Refer to see IETF [Deprecating TLS 1.0 and TLS 1.1](https://datatracker.ietf.org/doc/rfc8996/) for ditails.
+
+### Does TiDB Dedicated Tier support two-way TLS authentication between my client and TiDB Dedicated Tier cluster?
+
+No.
+
+Currently,TiDB Dedicated Tier support one-way TLS authentication, which means your client uses public key to verify the signature of your TiDB Dedicated Tier Cluster Server certificate's private key,while the server does not validate the client.
+
+### Does TiDB Dedicated Tier cluster have to configure TLS to establish a secure connection?
+
+Yes.
```

Comment:
Dedicated allow non-TLS connections.

Reply:
you are right, maybe this description should be updated after fix this issues , we will delete this content first.

Reply:
For this question, do we need to update "Yes" to "No" for now?

Reply:
updated this question